### PR TITLE
Add reset command to ectool

### PR DESCRIPTION
**Pull request by 13r0ck**

### CODE REVIEW

File: tool/src/main.rs [hunk 1]
```diff
@@ -601,6 +602,13 @@ fn main() {
                 process::exit(1);
             },
         },
+        Some(("reset", _sub_m)) => match unsafe { ec.reset() } {
+            Ok(()) => (),
+            Err(err) => {
+                eprintln!("failed to save LED settings: {:X?}", err);
```
Comment (crawfxrd) [inline tool/src/main.rs:608]: Message needs update

Comment (jackpot51) [inline tool/src/main.rs:608]: Wrong error message here.

Comment (13r0ck) [reply tool/src/main.rs]: thanks

Comment (13r0ck) [reply tool/src/main.rs]: thanks

**Review by jackpot51 — DISMISSED** [review]

File: tool/src/main.rs [hunk 2]
```diff
@@ -601,6 +602,13 @@ fn main() {
                 process::exit(1);
             },
         },
+        Some(("reset", _sub_m)) => match unsafe { ec.reset() } {
+            Ok(()) => (),
+            Err(err) => {
+                eprintln!("failed to send reset command: {:X?}", err);
```
Comment (crawfxrd) [inline tool/src/main.rs:608]: Is it only *sending* the command that will fail? What about: "failed to reset device"

**Review by crawfxrd — APPROVED** [review]

**Review by XV-02 — APPROVED** [review]
Brock mentioned that this was for exposing the Launch bootloader. Running it with the command he indicated (`ec --access hid reset`) worked, and running it haphazardly elsewhere didn't break anything, so looks good to me.

**Pull request merged**